File: Share/src/main/java/com/buguagaoshu/share/domain/ConfigData.java
package com.buguagaoshu.share.domain;

import jakarta.persistence.*;
import lombok.Data;

/**
 * @author Pu Zhiwei {@literal puzhiweipuzhiwei@foxmail.com}
 * @create 2024-06-28
 */
@Data
@Entity
@Table(name = "config_table")
public class ConfigData {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String name;

    private String configMessage;

    private Integer type;

    private Long createTime;

    private Long updateTime;
}

File: Go/internal/model/config_data.go
package model

// ConfigData 对齐 Spring com.buguagaoshu.share.domain.ConfigData（表 config_table）
type ConfigData struct {
	ID            *int64 `gorm:"primaryKey;autoIncrement;column:id" json:"id"`
	Name          string `gorm:"column:name" json:"name"`
	ConfigMessage string `gorm:"column:config_message" json:"configMessage"`
	Type          *int   `gorm:"column:type" json:"type"`
	CreateTime    *int64 `gorm:"column:create_time" json:"createTime"`
	UpdateTime    *int64 `gorm:"column:update_time" json:"updateTime"`
}

func (ConfigData) TableName() string { return "config_table" }
